tests/rhtap-demo/config/scenarios.go

```diff
@@ -29,7 +29,7 @@ var TestScenarios = []TestSpec{
 				GitSourceContext:           "",
 				GitSourceDefaultBranchName: "main",
 				HealthEndpoint:             "/",
-				SkipDeploymentCheck:        false,
+				SkipDeploymentCheck:        true,
```

Comment:
not super important, but should we perhaps remove the field `SkipDeploymentCheck` from `ComponentSpec` struct, since we no longer will be using that?

Reply:
I was about to remove it but then I thought we might resume checking the deployments later as part of some new feature so we can use the flag again someday in the future.

But I can remove it if you agree as I have no strong opinions here.

Reply:
ok let's keep it and not block this PR from merging